src/test/java/com/solr/clientwrapper/domain/service/SolrSearchRecordsServiceTest.java

```diff
@@ -0,0 +1,278 @@
+/**
+ * 
+ */
+package com.solr.clientwrapper.domain.service;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+
+/*import org.apache.solr.client.solrj.SolrServer;
+import org.apache.solr.client.solrj.SolrServerException;
+import org.apache.solr.client.solrj.impl.CommonsHttpSolrServer;*/
+
+import org.apache.solr.client.solrj.SolrClient;
+import org.apache.solr.client.solrj.SolrServerException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import com.solr.clientwrapper.domain.dto.solr.SolrAPIAdapterResponseDTO;
+import com.solr.clientwrapper.domain.dto.solrsearch.SolrSearchResponseDTO;
+import com.solr.clientwrapper.domain.port.api.SolrSearchRecordsServicePort;
+import com.solr.clientwrapper.infrastructure.adaptor.SolrAPIAdapter;
+
+@ExtendWith(MockitoExtension.class)
+@ExtendWith(SpringExtension.class)
+//@ActiveProfiles("test")
+@TestPropertySource(
```

Comment:
Rather than doing this, you can add these in application.yml  of test package

Reply:
Yeah, I tried that. There was some issue.., but will try again, sure.

Reply:
Sure, Let me know if you need any help :)

Reply:
Yeah, I tried from my side.. it looks like a complex process to me.. Let's discuss when you're free